test/resources/entity/with-endpoint.js

```diff
@@ -1,7 +1,6 @@
 import {Entity} from '../../../src/entity';
-import {resource, endpoint} from '../../../src/aurelia-orm';
+import {endpoint} from '../../../src/aurelia-orm';
 
-@resource()
```

Comment:
that's why i said: http://aurelia-orm.spoonx.org/decorators.html#endpoint
it doesn't there